ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -19,6 +19,10 @@ base_images:
     name: "4.3"
     namespace: ocp
     tag: oc-rpms
+  mco-rpms:
+    name: "4.3"
+    namespace: ocp
```

Comment:
```suggestion
    namespace: fcos```

Reply:
No, they should stay in ocp.

Reply:
CI is part of "ocp" and then flows out to "okd".  That will be one of the last things to change in this sequencing.

Reply:
would it make sense to add the above comment into the yaml file ? i think would help others to _understand_ the flow